docs/_static/manual_images/latest_versions_for_rtd_csv.csv

```diff
@@ -0,0 +1,5 @@
+Module Family,Latest Speed Version,Latest Servo Version,Latest Pulsing Version
+23-XX,0.0.32,0.0.21,0.1.0
+40-XX,0.1.0,0.0.0,\-
+60-XX,0.1.0,\-,\-
+81-XX,0.1.0,0.0.4,\-
```

Comment:
One quibble here may be that v0.0.4 of servo is around for 81G1s, but not G2s. Haven't gotten the servos synced up yet. May be worth listing the G2s and G1s separately on here or putting multiple versions and noting which is for which, or something to indicate what the deal is with their current versioning